vercel.json

```diff
@@ -633,6 +633,26 @@
       "source": "/docs/start-building/native-mobile-app",
       "destination": "https://www.ory.sh/docs/getting-started/integrate-auth/react-native",
       "permanent": false
+    },
+    {
+      "source": "docs/kratos/concepts/identity-schema",
+      "destination": "https://www.ory.sh/docs/guides/manage-identities/overview",
+      "permanent": false
+    },
+    {
+      "source": "docs/guides/manage-identities/managing-users-identities",
+      "destination": "https://www.ory.sh/docs/guides/manage-identities/create-identities",
+      "permanent": false
+    },
+    {
+      "source": "docs/guides/manage-identities/identity",
+      "destination": "https://www.ory.sh/docs/guides/manage-identities/identity-schema",
+      "permanent": false
+    },
+    {
+      "source": "docs/guides/manage-identities/managing-users-identities-metadata",
+      "destination": "https://www.ory.sh/docs/guides/manage-identities/identity-metadata-traits",
```

Comment:
For SEO it is usually a good idea to have "user", "identity" and "management" in the URL. This will rank it higher when people search for "user/identity management open source" for example. I don't think many people look for "identity traits open source"

Reply:
I'm not SEOing document IDs, I'm trying to come up with filenames+IDs that accurately represent the content. 
Please, let's keep SEO out of this :D

Reply:
Good SEO is paramount for documentation, it helps drive new users to Ory, makes pages easy to find on Google (which is what most users use to find content if they are facing an issue). Keeping good SEO practices in documentation is non-negotiable.

Reply:
Here are some resources to educate yourself on SEO and documentation and tech writing:

* https://docs.readthedocs.io/en/stable/guides/technical-docs-seo-guide.html
* https://idratherbewriting.com/2013/09/22/how-to-search-engine-optimize-your-help-content-or-documentation/ (very dated but premise remains the same)